resources/sdk/pureclouddotnet/config.json

```diff
@@ -77,6 +77,18 @@
         }
       ],
       "compileScripts": [
```

Comment:
This is a bit of a weird place to add unit tests, but it seemed like the least intrusive change.
The other tests were handled here, but I think it would require builds to be done in a specific order to avoid breaking if I added the new test file here: https://github.com/purecloudlabs/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PureCloudDotNetClientCodegen.java#L81